### DISCUSSION TITLE
Themes: the colours for the "Midnight" themes are so similar that I cannot see a difference

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#34] [1.1 General Help and Discussion]
Dear Alex, dear Community,

I checked out the different themes. In WinEdt 11, there is hardly any difference between the "Midnight" colours (compared to WinEdt 10).

WinEdt 11: Midnight black:
![e49025dc8fad57e4cadfca10c569d02c_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/afbf3578-ae87-48f7-87b9-64e310d39a27)
WinEdt 11: Midnight blue:

![dacc08234a9fb0741642b5fb61879da8_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/bb34fd2e-dab6-4c59-af69-67ceabaaf86a)

WinEdt 11: Midnight green:
![f332fdd8f4f8989e385b00dd42de1a5b_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/60bb3be2-ed74-4a89-a83c-06d7350db5c6)

WinEdt 10: Midnight black:
![8039f07e4fd315d2624dd74633dc6585_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/cfa674d9-0835-4153-8735-4ac605000637)

WinEdt 10: Midnight blue
![c9e186b6fa201a9f3a5d37b552d337b0_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/ce98b84b-0162-4a09-8c9b-4a793cbd5156)

inEdt 10: Midnight green:
![9af9dec5d379731f61407260625f819c_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/4cfe82f2-1454-44cd-ac4f-daa769ccff31)

WinEdt version:  64-bit and Build: 20240319  (v. 1.2) - 64-bit (portable)
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
I am rather blind and cannot see the difference or make decisions like this.

In the Themes submenu you will find an item "Edit Current Theme". You can experiment with the background color if you want them to be more or less saturated:

  clWindow="#121228"
  clWindowText="#E0E0E0"

If you come with perfect colors for these themes you can send them to me and I will replace the defaults. ClWindow is all you want to change. I don't use these themes so I don't know when they are too neutral and when they start giving you a headache. It depends...

**Reply by Alex-Simonic** [reply]
I experimented with background colors for Blue and Green "Midnight" Theme and I have to admit that you have a point. The appearance depends on the quality of the display and the ambiance light but they are too neutral and the blue hue is almost undetectable (especially in bright light). This has now been modified in the default settings by changing it to:

```
  clWindow="#000432" // Blue hue
  clWindow="#002008" // Green hue
```
Here is what you should do if you want to modify the colors for an active theme (or create a new one).:

In the Themes submenu execute "Edit Current Theme" (it is only enabled when you are not using the default Theme). This opens the theme. If this is one of the (read-only) predefined themes in `%B\ConfigEx\Themes` folder right-click on the document tab and select "Create Local Instance". Now you can modify and reload (Ctrl+F9) the local instance of this theme. You can increase or reduce the Blue or Green RGB (00..FF) value until you get the background that suits your eyes and your display. Double-clicking on clWindow will start the Colors GUI Dialog but I find it easier and more accurate to adjust RGB manually. The local theme will automatically become available in the list of available themes. In that list interface, you can double-click on any theme to load it without closing the dialog. This way you can switch and compare Black, Blue, and Green to detect the difference (or lack thereof).

This functionality has been significantly improved in WinEdt 11 (it does not exist in older versions)! The actual background colors may look different on different displays (for different eyes). That's why they can be easily adjusted as described above...
